pallets/automation-time/src/lib.rs

```diff
@@ -460,6 +566,28 @@ pub mod pallet {
 			10_000
 		}
 
+		fn transfer(
+			sender: T::AccountId,
+			recipient: T::AccountId,
+			amount: BalanceOf<T>,
+			task_id: T::Hash,
+		) -> Weight {
+			// check for sufficient funds in the sender Balance
+			let sender_balance = <T as Config>::Currency::free_balance(&sender);
+			if sender_balance < amount {
```

Comment:
are we sure the Currency::transfer won't check for this?

Reply:
![image](https://user-images.githubusercontent.com/22332240/154120247-d7d80233-1ea6-4cfe-9e71-8d2ef999bf49.png)
it will. but still, the problem is that it gives a generic `automationTime.TransferFailed` error. although maybe we're ok with that? i wanted to give a more specific `InsufficientFunds` error

Reply:
is the additional clarity worth the calculation time?